### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by smaxwell0120** [sergiocorreia/reghdfe#262]
Hi Sergio,

Is it not allowed to use the parallel computing option while including heterogeneous slope terms in the absvar? 
I am running a fixed effect model with a county fixed effect and a county group level time fixed effect. I also want to include a state-specific linear trend, so what I did is absorb(county, countypair#year state##year). When I add the parallel option, the parallel threads fail and the error code is 3301.

The absvar specification is fine without the parallel option. Can you help me figure out the problem?

**Comment by sergiocorreia** [comment]
Hi Dongning,

yes, the option should work. Can you share a toy example so I can try to replicate what's going on? e.g. something using the "sysuse auto" dataset?

**Comment by smaxwell0120** [comment]
@sergiocorreia yes, please see the example below. I used a different example dataset as the sysuse auto dataset is not a panel.

```
webuse nlswork

xtset idcode year

reghdfe ln_wage hours i.collgrad ttl_exp, a(birth_yr i.grade#year i.race##c.year) vce(cluster race) parallel(2,cores(2))
